### PR TITLE
Result

### PR DESCRIPTION
**Pull request by harshalini**
Added result page in quiz app.
It displays the player's score, the correct answers, and the answers selected by the player.

Preview -
https://62ce712504d6124d3696166a--snazzy-twilight-fe621b.netlify.app/

### CODE REVIEW

**Review by yashraj2312 — APPROVED** [review]
Looks good to merge

File: src/pages/result.js [hunk 1]
```diff
@@ -1,11 +1,60 @@
 import { useQuizQues } from "../context/quizQue-context"
+import { AiFillLinkedin, AiFillGithub, AiFillTwitterCircle } from "react-icons/ai"
+import { Link } from "react-router-dom";
+import { ExternalLink } from "react-external-link";
```
Comment (sushil-mod) [inline src/pages/result.js:4]: this was new for me, sure I will explore more on this

**Review by sushil-mod — APPROVED** [review]
The result page looks nice, and functionality is also working fine. Nice work

**Comment by harshalini** [comment]
> The result page looks nice, and functionality is also working fine. Nice work

Thanks!

**Pull request merged**